### PR TITLE
move grpc-1.0 and grpcClient-1.0 to GA

### PR DESCRIPTION
**Pull request by wtlucy**
For #14060

### CODE REVIEW

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.grpcClientSecurity-1.0.feature [hunk 1]
```diff
@@ -6,5 +6,5 @@ IBM-Provision-Capability: \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.ssl-1.0))"
 IBM-Install-Policy: when-satisfied
 -bundles=io.openliberty.grpc.1.0.internal.client.security
-kind=beta
+kind=ga
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.grpcClientSecurity-1.0.feature:9]: `WLP-Activation-Type: parallel` is needed on auto features too

Comment (wtlucy) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.grpcClientSecurity-1.0.feature:9]: I'd added it to the monitor autofeature but missed this one!

**Comment by wtlucy** [comment]
#build

**Review by billw1024 — APPROVED** [review]
yee-haw!

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#run-libby-bot

**Pull request merged**